docs/src/tutorials/params.md

```diff
@@ -94,4 +94,19 @@ for (j, sol) in enumerate(sols)
     gif(anim, "plot_$j.gif",fps=10)
 end
 
-```
\ No newline at end of file
+```
+
+## Scan parameter sets in parallel
+
+This toy example solves quickly by simply looping through parameter sets, but slower problems can benefit from parallel processing using an [EnsembleProblem](https://diffeq.sciml.ai/stable/features/ensemble/s).
+
+```
+Dnval = rand(1000)
+Dpval = rand(1000)
+
+function prob_func(prob,i,repeat)
+    remake(prob, p=[Dnval[i],Dpval[i]])
```

Comment:
You should probably add an index check https://mtk.sciml.ai/dev/basics/FAQ/#Getting-the-index-for-a-symbol in case it's reordered.

Reply:
Without having dug around inside `remake`, is this something that could be handled internally there to prevent reordering? Seems dangerous to have exposure to invisible bugs like this that users wouldn't foresee unless they crossed this FAQ.

Reply:
I'm not sure. Right now parameters never reorder, so technically this is correct. And with states, the reordering comes with reduction, so just passing an array of the right size will throw an error. But this is something we need to be working on. If an ODEProblem has a symbolic base, we should probably require a `force=true` or something to allow raw arrays to remake.

Reply:
I don't think we should have that. If one does not want reordering, then just don't use structural_simplify. Reordering is one of the most fundamental assumptions we have in MTK. Otherwise, there is not much point in giving states names other than for convenience.

Reply:
Well we'd need a `force=true` internally anyways, so it would make sense to add a safety check and assume people will be using transformed systems.

Reply:
Seems to me that if you are defining your problem symbolically, and therefore are using named states and `structural_simplify`, then `remake` should require a `Dict`/`Vector{Pair}`  for `ps` as in the system definition. This would also enhance consistency of the interface.

Reply:
`remake` should now work here with the pairs?